src/openrct2-ui/windows/Options.cpp

```diff
@@ -2180,11 +2180,9 @@ static void window_options_paint(rct_window* w, rct_drawpixelinfo* dpi)
             // Apply vertical alignment if appropriate.
             int32_t widgetHeight = pathWidget.bottom - pathWidget.top;
             int32_t lineHeight = font_get_line_height(gCurrentFontSpriteBase);
-            uint32_t padding = widgetHeight > lineHeight ? (widgetHeight - lineHeight) / 2 : 0;
-
-            gfx_draw_string_left_clipped(
-                dpi, STR_STRING, gCommonFormatArgs, w->colours[1], w->windowPos.x + pathWidget.left + 1,
-                w->windowPos.y + pathWidget.top + padding, 277);
+            int32_t padding = widgetHeight > lineHeight ? (widgetHeight - lineHeight) / 2 : 0;
+            ScreenCoordsXY screenCoords = { w->windowPos.x + pathWidget.left + 1, w->windowPos.y + pathWidget.top + padding };
```

Comment:
```suggestion
            ScreenCoordsXY screenCoords = { w->windowPos.x + pathWidget.left + 1, w->windowPos.y + pathWidget.top + static_cast<int32_t>(padding) };
```
Do this instead to avoid changing the padding type.

Reply:
ok thanks!